lib/resources/command.rb

```diff
@@ -28,6 +28,9 @@ def initialize(cmd)
     end
 
     def result
+      if @command.nil?
+        raise 'InSpec `command` was called with `nil` as the argument. This is not supported. Please provide a valid command instead.'
```

Comment:
Lets make it skip the command resource for now until we have proper exception handling